src/index.js

```diff
@@ -0,0 +1,168 @@
+"use strict";
+
+const state = {
+    city: 'Seattle',
+    temp: 0,
+    latitude: 0,
+    longitude: 0,
+};
+
+const getCityWeather = async (city) => {
+    await axios.get('http://127.0.0.1:5000/location', {
+        params: {
+            q: state.city,
+            format: 'json'
+        },
+    })
+    .then((response) => {
+        state.latitude = response.data[0]['lat'];
+        state.longitude = response.data[0]['lon'];
+        
+    })
+    .catch((error) => {
+        console.log('Oh haiil no, this does not exist.');
+        console.log(error.response);
+    });
+
+    getWeather();
+
+};
+
+const getWeather = async () => {
+    console.log(getWeather)
+    await axios.get('http://127.0.0.1:5000/weather', {
+        params: { 
+            lat: state.latitude,
+            lon: state.longitude,
+        }
+    }).then((response) => {
+        let realTemp = response.data.main.temp;
+        state.temp = Math.round((realTemp - 273) * 1.8 + 32);
+        document.querySelector('#tempValue');
+        tempChange();
+    })
+
+};
+
+
+const tempChange = () => {
+    let temp = state.temp;
+    let landscape = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    let color = 'red';
```

Comment:
```suggestion
    const temp = state.temp;
    let landscape;
    let color;
```
Couple small changes here - I'd recommend initializing these variables without a value so it's clear that their value will be set later instead of giving them arbitrary values that will be overwritten. Second, I'd actually recommend just referencing `state.temp` directly in your conditional statements rather than creating another `temp` variable that holds the same value. However, if you are going to have this variable, I'd recommend declaring it as a `const` variable since it shouldn't be modified.